crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -412,10 +412,16 @@ impl FromWorld for MeshPipeline {
                     count: None,
                 },
             ];
+
+            // EnvironmentMapLight
+            let environment_map_entries =
+                environment_map::get_bind_group_layout_entries([11, 12, 13]);
```

Comment:
I really like this pattern, we should probably do this for other parts of the engine. The prepass should probably do that at least.